README.md

```
@@ -1,17 +1,18 @@
 [![Coverage Status](https://coveralls.io/repos/github/vbpf/prevail/badge.svg?branch=main)](https://coveralls.io/github/vbpf/prevail?branch=main)[![CodeQL](https://github.com/vbpf/prevail/actions/workflows/codeql-analysis.yml/badge.svg?branch=main)](https://github.com/vbpf/prevail/actions/workflows/codeql-analysis.yml)
 
 # PREVAIL - A new eBPF verifier
-## a Polynomial-Runtime EBPF Verifier using an Abstract Interpretation Layer
-
 
+## a Polynomial-Runtime EBPF Verifier using an Abstract Interpretation Layer
 
-The version discussed in the [PLDI paper](https://vbpf.github.io/assets/prevail-paper.pdf) is available [here](https://github.com/vbpf/prevail/tree/d29fd26345c3126bf166cf1c45233a9b2f9fb0a0).
+The version discussed in the [PLDI paper](https://vbpf.github.io/assets/prevail-paper.pdf) is
+available [here](https://github.com/vbpf/prevail/tree/d29fd26345c3126bf166cf1c45233a9b2f9fb0a0).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)